src/01-functions.js

```diff
@@ -112,7 +124,14 @@ function averageThree() {}
  *  compareTypes("left", 10);
  *  //> false
  */
-function compareTypes() {}
+function compareTypes(a, b) {
+  if (typeof a === typeof b){
+    return true;
+  } else {
+    return false;
+  }
+  
```

Comment:
Here on 133 we should remove the empty line.  We use whitespace only when we need to separate code based on logic breaks.